adobecom/milo
# MWPW-119080 [Core] Languages & Locales

- Added the same prefix, language (itef) and typekit (tk) settings as is currently available on https://business.adobe.com.

Resolves: [MWPW-119080](https://jira.corp.adobe.com/browse/MWPW-119080)
To test: 
- Switch the language prefix in the urls to see the language (itef) get applied to the `html` `lang` attribute
- use browser's `view page source` , search for `typekit` to see the applied typekit link.


**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/[language_prefix]
- After: https://MWPW-119080--milo--adobecom.hlx.page/[language_prefix]
